src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final int ONE_ARRAY_INDEX = 1;
+
+    public WorkWithFile() {
+    }
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String statistic = readFromFile(fromFileName);
+        writeToFile(toFileName, statistic);
+    }
+
+    private void writeToFile(String toFileName, String statistic) {
+        File myFile = new File(toFileName);
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(myFile))) {
```

Comment:
myFile is redundant here. You can pass toFileName para to the FileWriter constructor directly